spec/src/main/asciidoc/chapters/applications.asciidoc

```diff
@@ -56,4 +56,31 @@ i.e., if a subclass or implementation method has any MVC or JAX-RS annotations t
 
 [tck-testable tck-id-class-vs-iface]#Annotations on a super-class take precedence over those on an implemented interface#. 
 The precedence over conflicting annotations defined in multiple implemented interfaces is implementation dependent. 
-Note that, in accordance to the JAX-RS rules, inheritance of class or interface annotations is not supported. 
+Note that, in accordance to the JAX-RS rules, inheritance of class or interface annotations is not supported.
+
+[[configuration_in_mvc]]
+Configuration in MVC
+~~~~~~~~~~~~~~~~~~~~
+
+Implementations MUST support configuration via the native JAX-RS configuration mechanism but MAY support other configuration sources.
+
+There are concrete configurations, that all MVC the implementations are `REQUIRED` the support such as : 
+
+- `ViewEngine.VIEW_FOLDER` 
+- `Csrf.CSRF_PROTECTION`
+
+Here's a simple example of how we can configure a custom location for the display folder other than the `/WEB-INF/views`, simply by overwriting the `getProperties` method of the subclass `Application`:
```

Comment:
Two notes:
  * how **you** can configure
  * custom location for the **view** folder

Sorry, I think I missed that one in my first review.

Reply:
Hi, 

I did not get it right, but the information that the user can change and configure is below.

Reply:
Sorry, I was not clean enough. My comment was about adjusting the sentence and use the bold words instead.

```
Here's a simple example of how you can configure a custom location for the view folder 
other than the `/WEB-INF/views`, simply by overwriting the `getProperties` method of the 
subclass `Application`:
```

Reply:
Does this whole sentence have to be bold?

Reply:
No, please don't make the words bold. I just made them bold to clarify which words I actually changed.

Reply:
ok .

Reply:
done.